### PR TITLE
Model11/ppm 003 004 project crud enhance

### PR DESCRIPTION
**Pull request by MinhBang1000**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -20,7 +20,8 @@
 		<module>ppm-infrastructure</module>
         <module>ppm-application</module>
 		<module>ppm-main</module>
-	</modules>
```
Comment (lenhuthao1908) [inline pom.xml:23]: Redundant code </modules>

File: ppm-domain/src/main/java/com/ctu/se/oda/model11/models/commands/requests/project/CreateProjectCommandRequest.java
Comment (lenhuthao1908) [inline ppm-domain/src/main/java/com/ctu/se/oda/model11/models/commands/requests/project/CreateProjectCommandRequest.java:1]: Code branch main have IDomainModel
If code branch not have IDomainModel, it cause a code conflict?

File: ppm-domain/src/main/java/com/ctu/se/oda/model11/models/commands/responses/project/CreateProjectCommandResponse.java
Comment (lenhuthao1908) [inline ppm-domain/src/main/java/com/ctu/se/oda/model11/models/commands/responses/project/CreateProjectCommandResponse.java:1]: Code branch main have IDomainModel
If code branch not have IDomainModel, it cause a code conflict?

File: ppm-infrastructure/src/main/java/com/ctu/se/oda/model11/repositories/IProjectRepository.java
Comment (lenhuthao1908) [inline ppm-infrastructure/src/main/java/com/ctu/se/oda/model11/repositories/IProjectRepository.java:1]: Should use type Long or UUID?

Comment (luuca792) [reply ppm-infrastructure/src/main/java/com/ctu/se/oda/model11/repositories/IProjectRepository.java:1]: UUID is the preferred type for Project entity's ID.  The line of code where Long type is declared is in RED color, which means it has been removed compared to the current code (of the main branch) and has been replaced by UUID.

**Review by lenhuthao1908 — APPROVED** [review]

**Pull request merged**